pkg/frontend/frontend.go

```diff
@@ -289,6 +289,12 @@ func (f *frontend) authenticatedRoutes(r *mux.Router) {
 	s.Methods(http.MethodGet).HandlerFunc(f.getAppLensDetector).Name("getAppLensDetector")
 
 	// Admin actions
+	s = r.
+		Path("/admin/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}/kubernetesobjects/force").
```

Comment:
open to input on naming/differentiation on the path

Reply:
/force implies something else IMO.  I would rather have a /restricted endpoint for the restricted one.

Reply:
I wanted to leave the current endpoint to ensure that everything that goes through it gets the restriction. Open to better names for the unrestricted endpoint.